docs/docs/guides/labs/dg/incrementally-adopting-dg/migrating-definitions.md

```diff
@@ -3,15 +3,21 @@ title: 'Autoloading existing Dagster definitions'
 sidebar_position: 100
 ---
 
+import Preview from '../../../../partials/\_Preview.md';
+
+<Preview />
+
 :::note
-This guide covers using existing Python definitions with dg. This guide presupposes a dg-enabled project. See the [Making a project dg-compatible](/guides/labs/dg/incrementally-adopting-dg/migrating-project) guide for more information.
+
+This guide covers using existing Dagster definitions with a `dg`-compatible project. To convert an existing project to use `dg`, see "[Converting an existing project to use `dg`](/guides/labs/dg/incrementally-adopting-dg/migrating-project)".
+
 :::
 
-In projects that heavily use `dg`, it's typical to keep all definitions underneath the `defs/` directory. However, if you've converted an existing project to use `dg`, you may have definitions located in various other modules. This guide will show you how to move these existing definitions into the `defs` directory in a way that will allow them to be automatically loaded.
+In projects that heavily use `dg`, you would typically keep all definitions in the `defs/` directory. However, if you've converted an existing project to use `dg`, you may have definitions located in various other modules. This guide will show you how to move these existing definitions into the `defs` directory in a way that will allow them to be automatically loaded.
 
-## Example project
+## Example project structure
 
-Let's walk through an example of how to migrating your existing definitions, with a project that has the following structure:
+Let's walk through an example of migrating your existing definitions, with a project that has the following structure:
 
 <CliInvocationExample path="docs_beta_snippets/docs_beta_snippets/guides/dg/migrating-definitions/1-tree.txt"  />
 
```

Comment:
I can't add a comment to that line, but this example is showing an `isort ignore` command.

```
<CodeExample path="docs_beta_snippets/docs_beta_snippets/guides/dg/migrating-definitions/2-definitions-before.py" title="my_existing_project/definitions.py" />
```
<img width="929" alt="image" src="https://github.com/user-attachments/assets/98e9cf79-739a-4a41-9e3c-80a4c025a916" />

Reply:
I'll add a `# start` comment to the snippet file, but will also make a ticket to add a function to filter `isort` comments to the CodeExample component, similar to `filterNoqaComments`.